CONTRIBUTING.md

```diff
@@ -0,0 +1,305 @@
+# How to contribute model to Open Model Zoo
+
+We appreciate your intention to contribute model to OpenVINO&trade; Open Model Zoo (OMZ). This guide would help you and explain main issues. OMZ is licensed under Apache License, Version 2.0. By contributing to the project, you agree to the license and copyright terms therein and release your contribution under these terms. Please note, that we accept models under permissive license: **MIT**, **Apache 2.0**, **BSD-3-Clause**. In other case it may take longer time to get approval (or even denial) for your model.
+
+Nowadays OMZ supports models from frameworks: 
+* Caffe\*
+* Caffe2\* (via conversion to ONNX\*)
+* TensorFlow\*
+* PyTorch\* (via conversion to ONNX\*)
+* MXNet\*
+
+## Pull request requirements
+
+Contribution to OMZ comes down to creating pull request (PR) in this repository. Please use `develop` branch when creating your PR. Pull request is strictly formalized and must contains:
+* configuration file  - `model.yml` (learn more in [Configuration file](#configuration-file) section)
+* documentation of model in markdown format (learn more in [Documentation](#documentation) section)
+* accuracy validation configuration file (learn more in [Accuracy Validation](#accuracy-validation) section)
+* license added to [tools/downloader/license.txt](tools/downloader/license.txt)
+* (*optional*) demo (learn more about it in [Demo](#demo) section)
+
+Name your model in OMZ using next rules:
+- name must be consistent with name given by authors, but full match not necessary
+- use lowercase
+- spaces are not allowed in the name, use `-` or `_` (`-` is preferable) as delimiters instead
+- suffix to model name, according to origin framework (see **`framework`** description in [configuration file](#configuration-file) section), if you adding reimplementation of existing model in OMZ from another framework
+
+This name will be used for downloading, converting, etc.
+Example:
+```
+resnet-50-pytorch
+mobilenet-v2-1.0-224
+```
+
+Configuration and documentation files must be located in `models/public/<model_name>` directory. 
```

Comment:
This section goes in some kind of Array of Structs fashion: 
- Pull request should contain configuration, documentation, validation, anddemo
- These are the places for configuration, documentation, validation and demo
- Tests should pass for configuration, documentation, validation and demo

I suggest to merge/remove/move the 2 latter "subsections"